setup.cfg

```diff
@@ -1,3 +1,64 @@
+[metadata]
+name = awkward
+url = https://github.com/scikit-hep/awkward-1.0
+project_urls =
+  Documentation = https://awkward-array.org
+  Source Code = https://github.com/scikit-hep/awkward-1.0
+  Bug Tracker = https://github.com/scikit-hep/awkward-1.0/issues
+  Discussions = https://github.com/scikit-hep/awkward-1.0/discussions
+  Chat = https://gitter.im/Scikit-HEP/awkward-array
+  Releases = https://github.com/scikit-hep/awkward-1.0/releases
+author = Jim Pivarski
+author_email = pivarski@princeton.edu
+maintainer = Scikit-HEP
+maintainer_email = scikit-hep-admins@googlegroups.com
+version = file: VERSION_INFO
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Intended Audience :: Developers
+    Intended Audience :: Information Technology
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Operating System :: MacOS :: MacOS X
+    Operating System :: Microsoft :: Windows
+    Operating System :: POSIX :: Linux
+    Operating System :: Unix
+    Programming Language :: Python
+    Programming Language :: Python :: 2.7
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Topic :: Scientific/Engineering
+    Topic :: Scientific/Engineering :: Information Analysis
+    Topic :: Scientific/Engineering :: Mathematics
+    Topic :: Scientific/Engineering :: Physics
+    Topic :: Software Development
+    Topic :: Utilities
+license = BSD 3-clause
+description = Manipulate JSON-like data with NumPy-like idioms.
+long_description = file: README-pypi.md
+long_description_content_type = text/markdown
+
+[options]
+python_requires = >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*
+packages = find:
+package_dir =
+    =src
+include_package_data = True
+zip_safe = False
+
+[options.packages.find]
+where = src
+exclude =
+    awkward_cuda_kernels
+
+[options.entry_points]
+numba_extensions =
+    init = awkward._connect._numba:register
```

Comment:
Just curious, how does numba handle name clashes here?

Reply:
I guess it's not a map, but more of an unordered list.

Reply:
I wonder that myself. The space of type names is a global namespace. When two versions of Awkward are accidentally loaded (possible through an old `awkward1` and a new `awkward`), the attempt to register another type with the same name causes an exception on Numba-loading time. I've been thinking they're opening themselves to complaints about other people's bugs by including this entry-point. On the other hand, I think that might be true of any entry-point: if someone writes a bad package that uses your entry-point, it would fail when _your_ package loads, making it look like your bug.